src/Arcus.Messaging.Tests.Unit/ServiceBus/AzureServiceBusMessagePumpOptionsTest.cs

```diff
@@ -43,6 +43,45 @@ public void TopicOptionsMaxConcurrentCalls_ValueIsNegative_ThrowsException()
             Assert.Throws<ArgumentException>(() => options.MaxConcurrentCalls = invalidConcurrentCalls);
         }
 
+        [Fact]
+        public void TopicOptionsPrefetchCount_ValueIsAboveZero_Succeeds()
+        {
+            // Arrange
+            var options = new AzureServiceBusMessagePumpOptions();
+            var validPrefetchCount = 42;
+
+            // Act
+            options.PrefetchCount = validPrefetchCount;
+
+            // Assert
+            Assert.Equal(validPrefetchCount, options.PrefetchCount);
+        }
+
+        [Fact]
+        public void TopicOptionsPrefetchCount_ValueIsZero_Succeeds()
+        {
+            // Arrange
+            var options = new AzureServiceBusMessagePumpOptions();
+            var validPrefetchCount = 0;
+
+            // Act
+            options.PrefetchCount = validPrefetchCount;
+
+            // Assert
+            Assert.Equal(validPrefetchCount, options.PrefetchCount);
+        }
+
+        [Fact]
+        public void TopicOptionsPrefetchCount_ValueIsNegative_ThrowsException()
+        {
+            // Arrange
+            var options = new AzureServiceBusMessagePumpOptions();
+            var invalidPrefetchCount = -1;
```

Comment:
Best that we use `Bogus` to generate our numbers here, instead of hard-code them.

Reply:
I can do that, but I'm wondering what would be the advantage of this in this specific situation ?

Reply:
Removing magic numbers, making the test intent clear (we're not interested in -1, we're interested in negative numbers), and removing edge-case mistakes.

Reply:
Changed this to make use of Bogus, but tbh, I don't see much added value for these specific cases.
Also saw that existing tests were using hardcoded values as well.